api/src/main/java/org/apache/cloudstack/api/command/admin/storage/SeedSystemVMTemplateCmd.java

```diff
@@ -0,0 +1,138 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.cloudstack.api.command.admin.storage;
+
+import java.util.HashSet;
+
+import org.apache.cloudstack.acl.RoleType;
+import org.apache.cloudstack.api.APICommand;
+import org.apache.cloudstack.api.ApiConstants;
+import org.apache.cloudstack.api.BaseCmd;
+import org.apache.cloudstack.api.Parameter;
+import org.apache.cloudstack.api.response.SeedSystemVMTemplateResponse;
+import org.apache.cloudstack.api.response.ZoneResponse;
+
+import com.cloud.user.Account;
+
+@APICommand(name = "seedSystemVMTemplate", description = "Copies a system vm template into secondary storage",
+        responseObject = ZoneResponse.class, requestHasSensitiveInfo = false, responseHasSensitiveInfo = false, since = "4.13",
+        authorized = {RoleType.Admin})
+public class SeedSystemVMTemplateCmd extends BaseCmd {
+
+    private static final String s_name = "seedsystemvmtemplateresponse";
+
+    /////////////////////////////////////////////////////
+    //////////////// API parameters /////////////////////
+    /////////////////////////////////////////////////////
+
+    @Parameter(name = ApiConstants.ID, type = CommandType.UUID, entityType = ZoneResponse.class, required = true, description = "the ID of the zone")
+    private Long id;
```

Comment:
I have (mental) "issues" with using "id" instead of "zoneid" as the parameter name, since it is indeed zone id... @Spaceman1984 can we please rename this to "zoneid"? /CC @PaulAngus @rhtyd @DaanHoogland

Reply:
Sure @andrijapanicsb, I'll make the change.